store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -145,18 +146,13 @@ func (h *rpcHandler) buildDAGExecutor(req *coprocessor.Request) (*dagContext, ex
 	if err != nil {
 		return nil, nil, nil, errors.Trace(err)
 	}
-	sc := flagsToStatementContext(dagReq.Flags)
 
-	// retrieving timezone by name first. When name is set, it means we need
-	// consider daylight saving time. If it is not, we can use offset.
-	if dagReq.TimeZoneName != "" {
-		if sc.TimeZone, err = LocCache.getLoc(dagReq.TimeZoneName); err != nil {
-			return nil, nil, nil, errors.Trace(err)
-		}
-		dagReq.TimeZoneName = sc.TimeZone.String()
```

Comment:
Should this be set?

Reply:
No, @zhexuany is removing it in another PR.